src/test/ui/issues/issue-22644.rs

```diff
@@ -1,5 +1,7 @@
+#![feature(type_ascription)]
+
 fn main() {
-    let a : u32 = 0;
+    let a : usize = 0;
```

Comment:
Why was this type changed? Isn't the fix _only_ to enable type_ascription?

Reply:
No, without this change the ascription causes type mismatch errors.